webextension/background.js

```diff
@@ -314,6 +314,10 @@ const portToPageAction = (function() {
       TabState.get().then(tabState => {
         // When the popup is hidden.
         updatePageActionIcon(tabState.tabId);
+        if (tabState.isShowingThankYouPage()) {
+          tabState.reset();
```

Comment:
Ah, yes, I just didn't spell out that this `else` clause is really for the cancel button:
```
      } else {
        closePageAction();
        tabState.reset();
        tabState.markAsVerified();
```

Maybe I should spell that out in the code just in case...

Reply:
CooL!